### PR TITLE
[ko] Update outdated files in dev-1.22-ko.1 (p3)

### PR DESCRIPTION
**Pull request by ClaudiaJKang**
Related Issue: #29257 

1. [X] M41.  `content/en/docs/reference/glossary/api-eviction.md`  | 1(+XS) 1(-)
1. [X] M42.  `content/en/docs/reference/glossary/kube-scheduler.md`  | 1(+XS) 1(-)
1. [X] M43.  `content/en/docs/reference/kubectl/kubectl.md`  | 24(+S) 0(-)
1. [X] M44.  `content/en/docs/reference/kubectl/overview.md`  | 1(+XS) 1(-)
1. [X] M46.  `content/en/docs/reference/scheduling/config.md`  | 67(+M) 59(-)
1. [X] M47.  `content/en/docs/reference/scheduling/policies.md`  | 1(+XS) 1(-)
1. [X] M48.  `content/en/docs/reference/tools/_index.md`  | 1(+XS) 1(-)
1. [X] M49.  `content/en/docs/reference/using-api/health-checks.md`  | 2(+XS) 2(-)
1. [X] M50.  `content/en/docs/setup/best-practices/cluster-large.md`  | 3(+XS) 0(-)

### CODE REVIEW

File: content/ko/docs/reference/glossary/api-eviction.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: API를 이용한 축출(Eviction)
 id: api-eviction
 date: 2021-04-27
-full_link: /docs/concepts/scheduling-eviction/pod-eviction/#api-eviction
+full_link: /docs/concepts/scheduling-eviction/api-eviction/
```
Comment (jihoon-seo) [inline content/ko/docs/reference/glossary/api-eviction.md:5]:
```suggestion
full_link: /ko/docs/concepts/scheduling-eviction/api-eviction/
```

그런데 이 부분은 `/ko` 를 붙여도 안 붙여도 별 영향이 없는 것 같기는 합니다.. 😊

**Review by jihoon-seo — COMMENTED** [review]
@ClaudiaJKang 고생이 많으셨습니다 😊
아래 comment는 검토해 보시고 선택적으로 반영하시면 되겠습니다. 😊
/lgtm

**Comment by jihoon-seo** [comment]
Previews:
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/reference/kubectl/kubectl/
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/reference/kubectl/overview/
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/reference/scheduling/config/
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/reference/scheduling/policies/
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/reference/tools/
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/reference/using-api/health-checks/
- https://deploy-preview-29826--k8s-dev-ko.netlify.app/ko/docs/setup/best-practices/cluster-large/

**Comment by ClaudiaJKang** [comment]
@jihoon-seo 반영했습니다~

**Comment by jihoon-seo** [comment]
@ClaudiaJKang 감사합니다~~ 😊
/lgtm

**Pull request merged**